src/main/scala/io/iohk/ethereum/faucet/Faucet.scala

```diff
@@ -0,0 +1,35 @@
+package io.iohk.ethereum.faucet
+
+import java.security.SecureRandom
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.stream.ActorMaterializer
+import io.iohk.ethereum.keystore.KeyStoreImpl
+import io.iohk.ethereum.mallet.service.RpcClient
+import io.iohk.ethereum.utils.{Config, Logger}
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import scala.util.{Failure, Success}
+
+object Faucet extends Logger {
+
+  def main(args: Array[String]): Unit = {
+    val config = FaucetConfig(Config.config)
```

Comment:
This makes faucet config dependent on mantis config. 
I suggest `FaucetConfig(ConfigFactory.load())` and `-Dconfig.file=./conf/faucet.conf` in `universal/bin/faucet-server`. Then we don't have to modify `mantis.conf` when running.